Watchman.Discord/GlobalSuppressions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
```

Comment:
czemu to powinno być w publicznym repo?

Reply:
Nie wiem czy to uznaje tylko Resharper czy czysty VS również, ale plik ten zawiera ukrywanie messages kompilatora, o nie potrzebnych zmiennych w niektórych konrolerach (DiscordRequest). Uznałem, że lepiej żeby wszystkim nie podkreślał tego jako błąd. Dużo to nie zmienia, więc obojętnie

Reply:
ok, może zostać 
ale nie jestem pewny czy Watchman.Discord to dobre miejsce na pliki tego typu
no chyba że musi tam być

Reply:
Miejsce akurat samo się wybrało, zobaczę czy da się przenieść

Reply:
Musi to być w jakimś projekcie, więc w sumie chyba i tak tutaj najlepiej